esp/src/src-react/components/WorkunitSummary.tsx

```diff
@@ -251,12 +251,10 @@ export const WorkunitSummary: React.FunctionComponent<WorkunitSummaryProps> = ({
                                     </MessageBar>
                                 }
                             </Sticky>
-                            <Sticky stickyPosition={StickyPositionType.Header}>
+                            <div style={{ position: "sticky", zIndex: 2, top: 44, display: "flex", flexDirection: "row", justifyContent: "space-between" }}>
```

Comment:
The position, z-index, and top rules are the replacement for the <Sticky>. The display flex, etc was just to make the summary content look a bit less strange as it scrolls beneath the WUID.

I'd not noticed this, but the <Sticky> seems to clone dom components on scrolling away (to force the sticky behavior) and then destroy them when you scroll back. And because the <WUStatus> has a useWorkunit hook in it, that constituted mounting / unmounting (I guess?) and was causing calls to WUQuery and WUInfo on every scroll.